### PR TITLE
feat: Adjust readme and contributing guide

### PR DESCRIPTION
**Pull request by RMCampos**
# Description

Update README and CONTRIBUTING.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

No tests required at this point.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

## Further comments

No comments.

### CODE REVIEW

**Review by mgaseta — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,43 +1,55 @@
-# Front-end Starting App
+[![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)](https://github.com/bcgov/nr-frontend-starting-app)
 
-This is a starting front-end app that intends to get the team started with code definitions, commits standards and guides related to code-style.
+# Natural Resources Front-End Starting PWA
 
-In case you're looking for the original readme template file, you can see it [here](README_template.md)
+This repository holds a set of policies, standard, guides and pipelines to get
+started with a PWA. Before writing your first line of code, please take a
+moment and check out our [CONTRIBUTING](CONTRIBUTING.md) guide.
```
Comment (annibalsilva) [inline README.md:7]: Suggestion: use the meaning of PWA (progressive web application, right?) in the header and again in the paragraph; _then_ we introduce the acronym. For instance:

```
# Natural Resources Front-End Starting Progressive Web Application

[...] to get stared with a progressive web application (PWA). [...]
```

And we can use _PWA_ from then on.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,55 @@
...
+  - We write code in TypeScript
+  - Redux!?
+  - Styled Components
+  - Jest and Cypress
+- Lint
+  - Airbnb ESLint!?
+- Tools
+  - Docker
+  - Visual Studio Code
+  - Postman
```
Comment (annibalsilva) [inline README.md:39]: Just checking, are we unsure about using Redux and the AirBnB lint?

File: README.md [hunk 3]
```diff
@@ -1,43 +1,55 @@
...
+## Our Policy
 
-## Stack
+- Work in the open: That means that everything we do should be open, should be
+public. Please, don't create private repositories unless you have a very strong
+reason. Keeping things public is a must follow rule for BC Government.
+- Customer centred services: All the work that's been created is to improve all
+of our users, customers, and friends lives. Is important to keep that in mind
+because as engineers sometimes we face technical issues, however, our goal is
+to have a nice and easy daily experience.
```
Comment (annibalsilva) [inline README.md:17]: Can we rephrase this? I'm unsure of this point's intent: is it saying the work we, the people responsible for this project, do here aims to make the lives of our customers and clients, *as well as* our own, better? Who's supposed to have an easy daily experience, the customers or us?

File: README.md [hunk 4]
```diff
@@ -1,43 +1,55 @@
...
 
-- React
-- Redux!?
-- Styled Components
-- Jest
-- Cypress
+# Stack
 
-## Getting started with the environment
+Here you will find a comprehensive list of all languages and tools that are
```
Comment (annibalsilva) [inline README.md:25]: We should have a "the" before "languages and tools", turning it into _list of all the languages and tools_.

I can't justify this one, though.

**Review by annibalsilva — CHANGES_REQUESTED** [review]
Documents are good, but some corrections are needed; I have a few suggestions as well.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,9 +1,66 @@
...
+If you like, Microsoft Visual Studio can be a great option. Lots of plugins
+and integrations. You can learn how to install [here](https://code.visualstudio.com/).
+
+### Check-style
+
+To be defined...
+
+## Run this application
+
+After setting up your environment you might want to see this service running.
```
[the remark this answers is not in the document]
Comment (RMCampos) [reply CONTRIBUTING.md:42]: Very nice! Thanks Annibal! Fixing.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,9 +1,66 @@
...
+
+## Follow our best practices
+
+- Typescript source code must be formatted according to
+[Google Java Style Guide](https://google.github.io/styleguide/javaguide.html),
+as mentioned. There's a pipeline to unsure all of our code is good to go.
+- We try to use [conventional commits](https://www.conventionalcommits.org/)
+because it makes the process of generating changelogs way easier. So we encourage
+you to read at least the [summary](https://www.conventionalcommits.org/en/v1.0.0/#summary)
+that summarize and give some examples.
```
[the remark this answers is not in the document]
Comment (RMCampos) [reply CONTRIBUTING.md:66]: Thanks for noticing! Fixed.

File: README.md (hunk 1 above)
Comment (RMCampos) [reply README.md:7]: Nice. Fixed.

File: README.md (hunk 2 above)
Comment (RMCampos) [reply README.md:39]: Yeah. Airbnb is defined. Redux probably yes.

File: README.md (hunk 3 above)
Comment (RMCampos) [reply README.md:17]: I'll fix it. Thanks for letting me know :)

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -35,18 +35,20 @@ and integrations. You can learn how to install [here](https://code.visualstudio.
 
 ### Check-style
 
-To be defined...
+To enforce a better solution and a stronger product we decided to use
+the Airbnb ESLint check-style. This way also hels us to have a dedicated
```
Comment (annibalsilva) [inline CONTRIBUTING.md:39]: hels -> helps

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -35,18 +35,20 @@ and integrations. You can learn how to install [here](https://code.visualstudio.
...
+Now that your environment is all set up, we can run the application. 
 You can get it up and running by typing `npm run dev` in the project
 root directory.
 
 ## Run tests
 
-You can run tests running `npm run test`. Tests coverage reposts can be seen
-on you command line window.
+You can run tests running `npm run test`. Tests coverage reports can be seen
+on you command line window and also on GitHub, in your commis and pull requests.
```
Comment (annibalsilva) [inline CONTRIBUTING.md:51]: commis -> commits

**Review by annibalsilva — DISMISSED** [review]
Just a few typos. Should be ready to be approved after they're corrected.

File: CONTRIBUTING.md (hunk 8 above)
Comment (RMCampos) [reply CONTRIBUTING.md:51]: Fixed. Thanks!

**Review by mgaseta — APPROVED** [review]

**Pull request merged**